app/src/main/res/values/styles.xml

```diff
@@ -13,15 +13,15 @@
         <item name="alertDialogTheme">@style/Theme.HomeAssistant.Dialog.Alert</item>
     </style>
 
-    <style name="TextAppearance.HomeAssistant.Headline" parent="TextAppearance.AppCompat.Headline">
+    <style name="TextAppearance.HomeAssistant.Headline" parent="TextAppearance.MaterialComponents.Headline1">
         <item name="android:textSize">20sp</item>
     </style>
 
-    <style name="TextAppearance.HomeAssistant.Title" parent="TextAppearance.AppCompat.Title">
+    <style name="TextAppearance.HomeAssistant.Title" parent="TextAppearance.MaterialComponents.Headline2">
         <item name="android:textSize">18sp</item>
     </style>
 
-    <style name="TextAppearance.HomeAssistant.Medium" parent="TextAppearance.AppCompat.Medium">
+    <style name="TextAppearance.HomeAssistant.Medium" parent="TextAppearance.MaterialComponents.Headline3">
```

Comment:
There are also `Widget.HomeAssistant.Button.Colored`, `Widget.HomeAssistant.Button.Borderless` and `Theme.HomeAssistant.Dialog.Alert` in styles which extend from `AppCompat`. You should change them as well

Reply:
done

Reply:
Looks good now :)